Sources/Commands/UserToolchain.swift

```diff
@@ -139,6 +145,14 @@ public struct UserToolchain: Toolchain {
             throw Error.invalidToolchain(problem: "could not find `clang` at expected path \(clangCompiler.asString)")
         }
 
+        do {
+            let path = try Process.checkNonZeroExit(arguments: findArgs(utility: "xctest")).chomp()
+            try AbsolutePath.validate(pathString: path)
```

Comment:
This method was removed in https://github.com/apple/swift-package-manager/pull/1540, right? I think you need to rebase.